docs/ocf-ble.md

```diff
@@ -32,21 +32,27 @@ is advertising over BLE. The following steps will create that connection which
 will show up as a IP interface on the Linux box:
 (Note: You will likely have to run these commands as root)
 
-1. Enable 6LoWPAN module
+1. Install BLE adapter
+
+  Plug in a USB BLE adapter, unless your system has it built in. We've used the
+  [ASUS USB-BT400](http://a.co/0ZlqU5C) and the
+  [Plugable BT4.0 micro adapter](http://a.co/diQXq2S) successfully.
+
+2. Enable 6LoWPAN module
 
   ```
   modprobe bluetooth_6lowpan
   echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
   ```
 
-2. Reset HCI
+3. Reset HCI
 
   ```
   hciconfig hci0
   hciconfig hci0 reset
   ```
 
-3. Look for your Arduino 101 advertisement
+4. Look for your Arduino 101 advertisement
```

Comment:
If you rebuild the app reboot the device, you don't have to repeat steps 1-4, I assume, right?   The user can just start from the connect step below, so it might be good to mention that.